### DISCUSSION TITLE
Python How to turn off the bot

### DISCUSSION THREAD

**Discussion by Skublik-glek** [PrismarineJS/mineflayer#2836] [Q&A]
I need to end python procces after bot did all actions. I use bot.quit(), bot.end() and sys.exit(), but procces still work.

**Comment by W3bParserV2** [comment]
### Learn the python bases
**Have you tried the following syntax ? If yes show me the output error.**

```python
quit()
```

**Reply by Skublik-glek** [reply]
There is no output error. It just continue works. Only os._exit(0) can stop this procces but this function is bad

**Reply by W3bParserV2** [reply]
Can you send the code?
